### ISSUE TITLE
Error: Unable to resolve module `AccessibilityInfo` 

### ISSUE THREAD

**Issue by fantasy525** [facebook/react-native#20707]
## Environment
React Native Environment Info:
    System:
      OS: Windows 10
      CPU: x64 Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
      Memory: 2.10 GB / 7.87 GB
    Binaries:
      Node: 8.11.3 - D:\nodejs\node.EXE
      Yarn: 1.6.0 - D:\yarn\bin\yarn.CMD
      npm: 6.2.0 - D:\nodejs\npm.CMD

## package.json
```
 "dependencies": {
    "react": "16.4.1",
    "react-native": "0.56.0"
  },
  "devDependencies": {
    "babel-jest": "23.4.2",
    "babel-preset-react-native": "5",
    "jest": "23.5.0",
    "react-test-renderer": "16.4.1"
  },
```
## Description
First:
1`.react-native init AwesomeProject`
2.`cd AwesomeProject`
3.`react-native run-android`
Then :build successful
```
BUILD SUCCESSFUL

Total time: 38.331 secs
Running D:\sdk/platform-tools/adb -s WEYDU17515001353 reverse tcp:8081 tcp:8081
Starting the app on WEYDU17515001353 (D:\sdk/platform-tools/adb -s WEYDU17515001353 shell am start -n com.awesomeproject/com.awesomeproject.MainActivity)...
Starting: Intent { cmp=com.awesomeproject/.MainActivity }

```
My pad runs apk，But the screen is red，
## The error message on the screen is：
`  The development server returned response error code:500:... Unable to resolve module AccessibilityInfo`

![img_20180817_105013](https://user-images.githubusercontent.com/22516157/44245526-a3212b00-a20b-11e8-8ee4-1427d7543b6c.jpg)

## Reproducible Demo
Let us know how to reproduce the issue. Include a code sample, share a project, or     share an app that reproduces the issue using https://snack.expo.io/. Please follow the guidelines for providing a MCVE: https://stackoverflow.com/help/mcve

**Comment by kelset** [comment]
Duplicate of https://github.com/facebook/react-native/issues/19953
